libkbfs/folder_branch_ops.go

```diff
@@ -1962,6 +1961,18 @@ func (fbo *folderBranchOps) GetNodeMetadata(ctx context.Context, node Node) (
 	if err != nil {
 		return res, err
 	}
+	_, prefetchStatus, _, err :=
+		fbo.config.BlockCache().GetWithPrefetch(res.BlockInfo.BlockPointer)
+	if err != nil {
+		dbc := fbo.config.DiskBlockCache()
+		if dbc != nil {
+			_, _, prefetchStatus, _ = fbo.config.DiskBlockCache().Get(ctx,
+				fbo.id(), res.BlockInfo.ID)
+		}
+		// Swallow cache errors since we don't want to communicate
```

Comment:
Maybe it would be good to explicitly set `prefetchStatus = NoPrefetch` here if `err != nil`, so that we don't have to rely on the return parameter in an error case.

(Same thing below.)

Reply:
Done. I also factored it out because I found myself hitting compile errors due to copypasta.